site/templates/application/nav/user_logged_in.jet

```diff
@@ -26,6 +26,11 @@
         <li>
           <a class="s72-dropdown-item" href="{{ routeToPath("/devices.html") }}">{{i18n("nav_devices")}}</a>
         </li>
+        {{if config("account_show_plans") == "true" }}
```

Comment:
Looks like this config is enabled for around 65 clients at the moment so i guess we may want to consider if it should show the subscription page for all use cases. (i think most of those clients are VIP sites where they are invited to plans rather than the user signing up / subscribing so could be wierd in terms of context for the user).

Reply:
Thanks for the heads up. I will check this up with either Jono or Nev

Reply:
@l0ud0gg Thanks for that feedback. We've decided to go for a completely new config `account_show_subscriptions` instead of `account_show_plans`